deployments/helm/dra-example-driver/values.yaml

```diff
@@ -46,6 +46,8 @@ controller:
 
 kubeletPlugin:
   priorityClassName: "system-node-critical"
+  seamlessUpgrades:
+    enabled: false
```

Comment:
We could enable this for e2e tests in case that happens to flag any issues.

https://github.com/kubernetes-sigs/dra-example-driver/blob/9ba4490762833a8ccffff5bf6f4c42d59eb15279/test/e2e/setup-e2e.sh#L36